lib/manager/gradle-lite/index.ts

```diff
@@ -0,0 +1,9 @@
+import * as gradleVersioning from '../../versioning/gradle';
+
+export { extractAllPackageFiles } from './extract';
+export { updateDependency } from './update';
+
+export const defaultConfig = {
+  fileMatch: ['(^|/)gradle.properties$', '\\.gradle(\\.kts)?$'],
+  versioning: gradleVersioning.id,
```

Comment:
```suggestion
  versioning: gradleVersioning.id,
  enabled: false,
```
Disable by default, so users need to explicit opt-in